### DISCUSSION TITLE
Helm Chart Out Of Date?

### DISCUSSION THREAD

**Discussion by dechristopher** [requarks/wiki#6609] [Help / Questions] [answered]
Hello Wiki.js,

We'd love to use the latest version via the helm repo, but it appears that the latest version available is v2.2.0. Is there any ongoing effort to upgrade the chart to the latest release?

Thanks,
Andrew

**Comment by NGPixel** [comment, marked as answer]
That's the helm version number, not Wiki.js version...

You will get the latest version from chart v2.2.0.
